inference-engine/tests/functional/plugin/cpu/single_layer_tests/ctc_Loss.cpp

```diff
@@ -0,0 +1,245 @@
+    }
+    return outs;
+}
+
+class CTCLossLayerCPUTest : public testing::WithParamInterface<CTCLossLayerCPUTestParams>, virtual public SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(const testing::TestParamInfo<CTCLossLayerCPUTestParams>& obj) {
+        CTCLossShapeParams shapes;
+        bool preprocessCollapseRepeated;
+        bool ctcMergeRepeated;
+        bool unique;
+        ElementType fPrecision;
+        ElementType iPrecision;
+        std::tie(shapes, preprocessCollapseRepeated, ctcMergeRepeated, unique, fPrecision, iPrecision) = obj.param;
+        std::ostringstream results;
+        results << "IS=" << CommonTestUtils::partialShape2str({shapes.first}) << "_";
+        results << "TS=";
+        for (auto& shapes : shapes.second) {
+            for (auto& shape : shapes) {
+                size_t N = shape[0];
+                size_t T = shape[1];
+                size_t C = shape[2];
+                results << "{" << N << "," << T << "," << C << "}"
+                    << "_";
+            }
+        }
+        results << "preprocessCollapseRepeated=" << preprocessCollapseRepeated << "_";
+        results << "ctcMergeRepeated=" << ctcMergeRepeated << "_";
+        results << "unique=" << unique << "_";
+
+        results << "fPrecision=" << fPrecision << "_";
+        results << "iPrecision=" << iPrecision << "_";
+
+        return results.str();
+    }
+
+protected:
+    void SetUp() override {
+        CTCLossShapeParams shapes;
+        bool preprocessCollapseRepeated;
+        bool ctcMergeRepeated;
+        bool unique;
+        ElementType fPrecision;
+        ElementType iPrecision;
+        std::tie(shapes, preprocessCollapseRepeated, ctcMergeRepeated, unique, fPrecision, iPrecision) = GetParam();
+
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        selectedType = std::string("ref_any_FP32");
+
+        for (auto& shapes : shapes.second) {
+            for (auto& shape : shapes) {
+                size_t N = shape[0];
+                size_t T = shape[1];
+                size_t C = shape[2];
+                targetStaticShapes.push_back({{N, T, C}, {N}, {N, T}, {N}});
+            }
+        }
+
+        auto inputDynamicShapesValues = shapes.first.front();
+        ov::PartialShape shapeN{inputDynamicShapesValues[0]};
+        ov::PartialShape shapeNT{inputDynamicShapesValues[0], inputDynamicShapesValues[1]};
+        ov::PartialShape shapeNTC{inputDynamicShapesValues[0], inputDynamicShapesValues[1], inputDynamicShapesValues[2]};
+        inputDynamicShapes = {shapeNTC, shapeN, shapeNT, shapeN};
+
+        std::vector<ElementType> types{fPrecision, iPrecision, iPrecision, iPrecision};
+        std::vector<ov::PartialShape> partialShapes{inputDynamicShapesValues, shapeN, shapeNT, shapeN};
+
+        auto params = makeDynamicParams(types, partialShapes);
+        auto bank = ngraph::op::Constant::create(ngraph::element::i64, ngraph::Shape{ }, {7});
```

Comment:
maybe better make it test parameter?

Reply:
now the blank value is read from test parameters. Thanks!